/hub_motor_driver_v2:
  ros__parameters:
    acel_ms: 1200
    acel_ms_corr: 1200
    decel_ms: 1200
    decel_ms_corr: 1200
    use_sim_time: false
